webextensions/api/devtools.json

```
@@ -711,6 +1110,132 @@
               }
             }
           },
+          "openResource": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "16"
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            },
+            "columnNumber": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "114"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "16"
+                  },
+                  "safari_ios": {
+                    "version_added": false
+                  }
+                }
+              }
+            },
+            "lineNumber": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": true
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "16"
+                  },
+                  "safari_ios": {
+                    "version_added": false
+                  }
+                }
+              }
+            },
+            "url": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": true
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "16"
+                  },
+                  "safari_ios": {
+                    "version_added": false
+                  }
+                }
+              }
+            }
+          },
+          "setOpenResourceHandler": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
```

Added in [this commit before Chrome 16](https://github.com/chromium/chromium/commit/6b3858aecac0b03ff54a1700e5e997a8693715fa), when `devtools.panels` was still experimental:

```suggestion
                  "version_added": "18"
```